platform/src/main/java/org/hillview/table/EqualityFilter.java

```diff
@@ -0,0 +1,52 @@
+package org.hillview.table;
+
+import org.hillview.table.api.IColumn;
+import org.hillview.table.api.ITable;
+import org.hillview.utils.Converters;
+
+/**
+ * This filter maps a given Table to a Table that only contains the given value in the specified column.
+ * @param <T>
+ */
+public class EqualityFilter<T> implements TableFilter {
+    private String columnName;
```

Comment:
I used the `@Nonnull` annotation from `javax.annotations`. It that the one you meant?